src/System.Data.SqlClient/src/System/Data/SqlClient/SqlErrorCollection.cs

```diff
@@ -2,64 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
-
-//------------------------------------------------------------------------------
-
 using System.Collections;
-
+using System.Collections.Generic;
 
 namespace System.Data.SqlClient
 {
     public sealed class SqlErrorCollection : ICollection
```

Comment:
That'd have to be discussed in a separate issue and go through the [API Review Process](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md). It'd require similar changes to be added to a future version of the full framework.

Note that the existing `Add` method is `internal`, so if this did implement `ICollection<SqlError>`, it'd very likely implement it with `ICollection<SqlError>.IsReadOnly` returning `true` and throwing from `ICollection<SqlError>.Add` so no external callers can modify the collection.